site/content/blog/2021-04-01-whats-new-in-svelte-april-2021.md

```diff
@@ -0,0 +1,67 @@
+---
+title: What's new in Svelte: April 2021
+description: SvelteKit beta and new way to use slots
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Two projects that have been months (even years) in the making have made their way out into the world. SvelteKit is now in public beta and slotted components are now available in Svelte!
+
+## What's up with SvelteKit?
+SvelteKit - Svelte's meta-framework for building SSR, serverless applications - is now officially in public beta. Expect bugs! Lots more detail in the [latest blog post](https://svelte.dev/blog/sveltekit-beta). Want to know when 1.0 is close? Check out the milestone on [github](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
+
+## New in Svelte & Language Tools
+- Slotted components, including `<svelte:fragment slot="...">` lets component consumers target specific slots with rich content (**Svelte 3.35.0, Language Tools [104.5.0](https://github.com/sveltejs/language-tools/releases/tag/extensions-104.5.0)**, [documentation](https://svelte.dev/docs#slot_name))
```

Comment:
It might be good to note that this is currently not documented, though there's a PR open to add docs (#6097).

That PR has a pretty good description of the new feature:
> Components can be placed in a named slot using the syntax `<Component slot="name" />`.
In order to place content in a slot without losing the flow layout (ie: without using a wrapper element) you can use the special element `<svelte:fragment>`.

Reply:
Thanks! Do you think the docs will be merged by the first? If so, what anchor link do you think would make the most sense to point to here? Adding your explanation as a stop-gap just in case :)

Reply:
@dummdidumm just approved the PR a few hours ago, so maybe it will be merged in time?

If it does get merged, the docs on [named slots](https://svelte.dev/docs#slot_name) and [svelte:fragment](https://svelte.dev/docs#svelte_fragment) (not live yet), as well as the new tutorial on [svelte:fragment](https://svelte.dev/tutorial/svelte-fragment) (also not live) would be good to link.

Reply:
It's live now 
https://svelte.dev/tutorial/svelte-fragment 
https://svelte.dev/docs#slot_name
https://svelte.dev/docs#svelte_fragment